packages/manager/modules/vps/src/migration/vps-migration.html

```diff
@@ -0,0 +1,108 @@
+        </a>
+        <p data-translate="vps_migration_description2"></p>
+        <span data-translate="vps_migration_description3"></span>
+        <b><span data-translate="vps_migration_description_date"></span></b>
+        <p>
+            <b><span data-translate="vps_migration_description4"></span></b>
+        </p>
+        <p data-translate="vps_migration_description5"></p>
+        <p data-translate="vps_migration_description6"></p>
+    </div>
+    <oui-message data-type="info" class="migration-info">
+        <p data-translate="vps_migration_info1"></p>
+        <a
+            class="oui-link oui-link_icon"
+            data-ng-href="{{ $ctrl.externalUrl }}"
+            rel="noopener"
+            target="_blank"
+        >
+            <span data-translate="vps_migration_info2"></span>
+            <span class="fa fa-external-link" aria-hidden="true"></span>
+        </a>
+    </oui-message>
+    <oui-datagrid
+        class="mt-3"
+        data-rows="$ctrl.vpsList"
+        data-row-loader="$ctrl.getMigrationInfo($row)"
+        data-selectable-rows
+        data-on-row-select="$ctrl.onRowSelect($row, $rows)"
+    >
+        <extra-top>
+            <oui-button
+                class="mb-3"
+                data-type="button"
+                data-variant="secondary"
+                data-on-click="$ctrl.migrate($rows)"
+                data-disabled="!($ctrl.selectedRows && $ctrl.selectedRows.length > 0)"
+            >
+                <span data-translate="vps_migration_button"></span>
+            </oui-button>
+        </extra-top>
+        <oui-column
+            data-title="'vps_migration_table_detail' | translate"
+            data-property="displayName"
+            data-sortable
+        >
+            <div>
+                <b><span data-ng-bind="$row.migrationDescription"></span></b>
+                <br />
+                <a data-ui-sref="vps.detail({serviceName : $row.name})">
+                    <span data-ng-bind="$row.displayName"></span> -
+                    <span data-ng-bind="$row.description"></span>
+                </a>
+            </div>
+        </oui-column>
+        <oui-column
+            data-title="'vps_migration_table_status' | translate"
+            data-property="migration.status"
+            data-sortable
+        >
+            <span
+                data-ng-class="{
+                    'oui-status oui-status_error': $row.migration.status === 'toPlan',
+                    'oui-status oui-status_warning': !$row.migration.status === 'toPlan'}"
+                data-ng-bind="'vps_migration_status_' + $row.migration.status | translate"
+            >
+            </span>
+        </oui-column>
+        <oui-column
+            data-title="'vps_migration_table_planned_date' | translate"
+            data-property="migration.date"
+        >
+            <span
+                data-ng-if="$row.migration.date"
+                data-ng-bind="$row.migration.date | date:'mediumDate'"
```

Comment:
parse in locale date

Reply:
The browser's timezone should automatically be used. But we should check this once we get the sate from the API